src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Converters/EnumToBoolConverter_Tests.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Converters;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Converters
+{
+	public enum TestEnumForEnumToBoolConverter
```

Comment:
Will this work if the `enum` has the `Flag` attribute? Can we have a test for it?

Reply:
@pictos No. The current implementation does not support this. But is a good idea to support this.